src/Makefile

```diff
@@ -22,7 +22,10 @@
 ### ==========================================================================
 
 ### Establish the operating system name
-UNAME = $(shell uname)
+KERNEL = $(shell uname -s)
+ifneq ($(KERNEL),Darwin)
```

Comment:
Why do we need this "ifneq" clause? Can't we just always call "shell uname -o"? Or doesn't it work with Mac?